daemon/logger/jsonfilelog/jsonfilelog_test.go

```diff
@@ -54,6 +56,49 @@ func TestJSONFileLogger(t *testing.T) {
 	}
 }
 
+func TestJSONFileLoggerWithTags(t *testing.T) {
+	cid := "a7317399f3f857173c6179d44823594f8294678dea9999662e5c625b5a1c7657"
+	cname := "test-container"
+	tmp, err := ioutil.TempDir("", "docker-logger-")
+	if err != nil {
+		require.NoError(t, err)
+	}
+	defer os.RemoveAll(tmp)
+	filename := filepath.Join(tmp, "container.log")
+	l, err := New(logger.Info{
+		Config: map[string]string{
+			"tag": "{{.ID}}/{{.Name}}", // first 12 characters of ContainerID and full ContainerName
+		},
+		ContainerID:   cid,
+		ContainerName: cname,
+		LogPath:       filename,
+	})
+	if err != nil {
+		t.Fatal(err)
+	}
```

Comment:
`require.NoError(t, err)`

same with all the lines below that check for `err != nil`

Reply:
Done for the new method I added. I did not correct the rest of the file.